components/content/haunted-house/index.vue

```diff
@@ -0,0 +1,57 @@
+<script setup lang="ts">
+import { shallowRef } from 'vue'
+import { TresCanvas } from '@tresjs/core'
+import { OrbitControls, useTweakPane } from '@tresjs/cientos'
+import { PCFSoftShadowMap, sRGBEncoding, NoToneMapping, Vector3 } from 'three'
+import Floor from './floor.vue'
+import House from './house.vue'
+import Ghosts from './ghosts.vue'
+
+const { pane } = useTweakPane()
+
+const gl = {
+  clearColor: '#262837',
+  shadows: true,
+  alpha: false,
+  shadowMapType: PCFSoftShadowMap,
+  outputEncoding: sRGBEncoding,
```

Comment:
`outputEncoding` and it's options `sRGBEncoding` are deprecated since v152 of threejs 

You can replace it with `outputColorSpace: SRGBColorSpace,`

See more here  https://github.com/Tresjs/tres/pull/244/files